config.default.js

```diff
@@ -9,7 +9,7 @@ module.exports = function () {
     'fullscreen': true,
     'fullscreenFrame': true,
     'nodeAttr': [
-      // value can be a node attribute (1 depth) or a a function in utils/node with prefix show
+      // value can be a node attribute (depth 1) or any of the functions with a name starting with "show" from lib/utils/node.js with
```

Comment:
of cource the functions are in lib and ending is unimportant (not both not needed in a require)

Reply:
Ich denke, so erklärt der Kommentar deutlich, wie sich das `value` zusammensetzt, bzw. wo man die möglichen Werte findet, wenn man dabei ist, die config.default.js an seine Community anzupassen.